src/app/pages/storage/snapshots/store/snapshot.actions.ts

```diff
@@ -0,0 +1,12 @@
+import { createAction, props } from '@ngrx/store';
+import { ZfsSnapshot } from 'app/interfaces/zfs-snapshot.interface';
+
+export const loadSnapshots = createAction('[Snapshots API] Load', props<{ extra: boolean }>());
```

Comment:
Prefer actions as events, not commands. In this case, this should be snapshotPageEntered or something like this. Extra argument belongs elsewhere.